src/actwebsocket.js

```diff
@@ -28,6 +28,7 @@ export default function initActWebSocket() {
       )
     }
   }
+  return true
 }
 
```

Comment:
Why are we returning boolean?

Reply:
we need to identify if ACTWebSocket is in use to flip the order in a couple places, we don't want to perform regex (easiest way to check that is old chromium compatible) everytime a new render is requested and gone throguh the DPS so we pass it through here as a prop to components that need the information

Reply:
I think I didn't understand 😢 

I'm asking why did you change this specific function to return a boolean instead of void (no value or undefined). I understood you need to check if ACTWebSocket is in use but from what I could see it's not from the value of this function, right? Or is it?